### DISCUSSION TITLE
Recent Elasticsearch Changes Not Working Well

### DISCUSSION THREAD

**Discussion by ZigTic** [requarks/wiki#3536] [Error / Bug Report]
**Describe the bug**
I had been using the Elasticsearch module with MariaDB in Wiki.js 2.3 stable, and it was working great.  I upgraded to 2.4.107 today, and noticed the search results weren't sorting well.  For example, I could search for the exact title of a page, but the page I'm looking for would be 5 to 10 pages deep in the search results whereas before the upgrade it would have been the top result.

It seemed to be sorting purely alphabetically instead of ranking certain results and pushing them towards the top, as it had before.

For now, I've reverted back to the engine.js from the 2.3 stable build, and that has resolved the problem for me.

**Expected behavior**
Expecting the page I'm looking for to be at the top of my search results or in the first couple of pages.

**Host Info (please complete the following information):**
 - OS: CentOS 7
 - Wiki.js version: 2.4.107
 - Database engine: MariaDB 10.4.13
 - Elasticsearch 7.7.0

**Comment by NGPixel** [comment]
There was a single change made via this PR: https://github.com/Requarks/wiki/pull/1882
It might need to be improved to solve your sorting issue.

**Comment by Feder1co5oave** [comment]
@NGPixel There was also #2647, that might have solved the issue. It hasn't been released yet.
